components/datetime/tests/fixtures/tests/components.json

```diff
@@ -17,16 +17,16 @@
         },
         "output": {
             "values": {
-                "en": "Tuesday, January 21, 2020 Anno Domini at 08:25:07",
```

Comment:
Note: I know some of these tests were specifically trying to test the handling of patterns with escaped pattern characters, like `{0} 'at' {1}`, which is obviously quite different than `{0} at {1}` in a datetime pattern.

Can you verify that we still have such a test, and if not, add one?

Reply:
As far as testing escape sequences in the pattern goes, the file `components/datetime/tests/fixtures/tests/components-exact-matches.json` has a couple of tests for "yw => 'week' w 'of' Y" and "MMMMW -> 'week' W 'of' MMMM".